[35mDEBUG   [0m src.steps.common:common.py:18 Running fixture setup: start_1_node
[32mINFO    [0m src.node.status_node:status_node.py:39 Starting node with command: ['./status-cli', 'serve', '-n', 'first_node', '-p', '39434']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-02-25T00:25:18.734Z	INFO	status-cli/serve.go:34	Running serve command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -name first_node
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -port 39434
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -light false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -api-modules waku,wakuext,wakuv2,permissions,eth
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -telemetry-server-url
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -fleet status.staging
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-02-25T00:25:18.735Z	INFO	first_node	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-25|00:25:20.925] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-25|00:25:20.931] New local node record                    seq=1,740,443,120,929 id=d69fa6b9b46006fa ip=127.0.0.1 udp=0 tcp=36575
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-25|00:25:20.931] Started P2P networking                   self="enode://bbaa5a0f98f28027c31c7ac18c2ba2a9d3a878819cd59a29e9266fbd62af47f32888f10524fa7bf23637e380ec315a3565f864108d8a693795a521f6ed4b7d7f@127.0.0.1:36575?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - ERROR[02-25|00:25:20.932] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-25|00:25:20.933] HTTP server started                      endpoint=127.0.0.1:39434 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-25|00:25:20.959] New local node record                    seq=1,740,443,120,941 id=c96ac776770817a1 ip=10.1.0.178 udp=32967 tcp=34983
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-25|00:25:20.963] New local node record                    seq=1,740,443,120,942 id=c96ac776770817a1 ip=10.1.0.178 udp=32967 tcp=34983
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-02-25T00:25:21.394Z	INFO	first_node	status-cli/util.go:68	account created, key UID: 0x1310bab6f97edcadfb48f9c192e454dd64cd9c1528cf21f67eaf1629cde60d3b
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-25|00:25:21.395] New local node record                    seq=1,740,443,120,943 id=c96ac776770817a1 ip=10.1.0.178 udp=32967 tcp=34983
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-02-25T00:25:21.423Z	INFO	first_node	status-cli/util.go:93	messenger started, public key: 0x04cfa299d255ad2a2b211b19d2147c5ac28442382f723de3bbb41bd2b7cd6112a7784b5fa9ed684ceff00da9431fe6265740f07ba97eb04a554a71ef5482fade76
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - 2025-02-25T00:25:26.424Z	INFO	first_node	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130', '-kid', '0xd8d0a6ec6b7198cb0217115fd766fab8ff146a351ef79446c2dec6d7dbf7a007']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - 2025-02-25T00:25:26.740Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - -name 0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - -key-uid 0xd8d0a6ec6b7198cb0217115fd766fab8ff146a351ef79446c2dec6d7dbf7a007
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - 2025-02-25T00:25:26.740Z	INFO	0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:25:26.777] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - ERROR[02-25|00:25:26.785] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:25:26.785] New local node record                    seq=1,737,419,564,795 id=21e155c21669159a ip=127.0.0.1 udp=0 tcp=34727
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:25:26.785] Started P2P networking                   self="enode://3adb140cde1eb80c72ac2a83452dabbd2d236498950c66e2194aba96737679facd1c9dc1ec9a7a2052df0b0b89ad1b038868776452403d573e8f3551b0e26fbc@127.0.0.1:34727?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:25:26.786] HTTP server started                      endpoint=127.0.0.1:6130 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:25:26.816] New local node record                    seq=1,740,443,126,796 id=a6f2f22fe1d4bca5 ip=10.1.0.178 udp=0 tcp=38933
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:25:26.820] New local node record                    seq=1,740,443,126,797 id=a6f2f22fe1d4bca5 ip=10.1.0.178 udp=54602 tcp=38933
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:25:26.821] New local node record                    seq=1,740,443,126,798 id=a6f2f22fe1d4bca5 ip=10.1.0.178 udp=54602 tcp=38933
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - 2025-02-25T00:25:26.951Z	INFO	0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130	status-cli/util.go:62	existing account, key UID: 0xd8d0a6ec6b7198cb0217115fd766fab8ff146a351ef79446c2dec6d7dbf7a007
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:25:26.951] New local node record                    seq=1,740,443,126,799 id=a6f2f22fe1d4bca5 ip=10.1.0.178 udp=54602 tcp=38933
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - 2025-02-25T00:25:26.972Z	INFO	0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130	status-cli/util.go:93	messenger started, public key: 0x04c61f8fa3515a136d6a0860b6cdb68ecd4b2d8df3f494a1b628b5f159c8168eb431422a84d88654a8bcffcccf7130e656351c12b7f31eb621de6d944d9c62b38d
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137', '-kid', '0x19f84f473f249eb2a536e729d1f2c0efd07cf8c214901a0faa4d83f2e71ece36']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - 2025-02-25T00:25:28.742Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - -name 0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - -key-uid 0x19f84f473f249eb2a536e729d1f2c0efd07cf8c214901a0faa4d83f2e71ece36
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - 2025-02-25T00:25:28.742Z	INFO	0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-25|00:25:28.779] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-25|00:25:28.785] New local node record                    seq=1,737,419,633,056 id=4d486e8dbc7eb7b6 ip=127.0.0.1 udp=0 tcp=33719
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-25|00:25:28.786] Started P2P networking                   self="enode://88fdc7f7abca4ed0a72cfac7bbeb5ff6e6aeeb5dc2f2baf3f0e4dc9c1a9781862fa1544e09a59c772ded4335d9dbb9bc1e53eace961ebeb40bc9456f10dafa8f@127.0.0.1:33719?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - ERROR[02-25|00:25:28.786] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-25|00:25:28.787] HTTP server started                      endpoint=127.0.0.1:6137 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-25|00:25:28.824] New local node record                    seq=1,740,443,128,798 id=750017867e0995b4 ip=10.1.0.178 udp=0 tcp=41283
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-25|00:25:28.829] New local node record                    seq=1,740,443,128,799 id=750017867e0995b4 ip=10.1.0.178 udp=37743 tcp=41283
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-25|00:25:28.830] New local node record                    seq=1,740,443,128,800 id=750017867e0995b4 ip=10.1.0.178 udp=37743 tcp=41283
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - 2025-02-25T00:25:28.960Z	INFO	0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137	status-cli/util.go:62	existing account, key UID: 0x19f84f473f249eb2a536e729d1f2c0efd07cf8c214901a0faa4d83f2e71ece36
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-25|00:25:28.961] New local node record                    seq=1,740,443,128,801 id=750017867e0995b4 ip=10.1.0.178 udp=37743 tcp=41283
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - 2025-02-25T00:25:28.982Z	INFO	0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137	status-cli/util.go:93	messenger started, public key: 0x04f4a3da8d5133481b6c157534cf62275127bb04b5f4dd1fde013c583a2708bc40001ecc2ffbf4edcc4f03aab32c3f76de8ff673dc40d690ccda6d9dea3ada4f6c
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133', '-kid', '0xa9461a9f60364bb61116eabe5cc9b95fd07f7c283d742e3ad11dfd6dbe9b5d89']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:30.738Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - -name 0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - -key-uid 0xa9461a9f60364bb61116eabe5cc9b95fd07f7c283d742e3ad11dfd6dbe9b5d89
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:30.738Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-25|00:25:30.782] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - ERROR[02-25|00:25:30.788] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-25|00:25:30.788] New local node record                    seq=1,737,419,596,535 id=ceb3ac46208d5874 ip=127.0.0.1 udp=0 tcp=43405
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-25|00:25:30.788] Started P2P networking                   self="enode://258928b145e8e2cfa71d9f79b1c6107f8a42182ee831449209e212c28895554f9555253596f26c14c2f2aebc9bab14e51fea9341444f9f3da43e3fb66230428b@127.0.0.1:43405?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-25|00:25:30.789] HTTP server started                      endpoint=127.0.0.1:6133 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-25|00:25:30.817] New local node record                    seq=1,740,443,130,800 id=5c0f481e52bf6766 ip=10.1.0.178 udp=0 tcp=38977
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-25|00:25:30.818] New local node record                    seq=1,740,443,130,801 id=5c0f481e52bf6766 ip=10.1.0.178 udp=39529 tcp=38977
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-25|00:25:30.826] New local node record                    seq=1,740,443,130,802 id=5c0f481e52bf6766 ip=10.1.0.178 udp=39529 tcp=38977
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:30.960Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/util.go:62	existing account, key UID: 0xa9461a9f60364bb61116eabe5cc9b95fd07f7c283d742e3ad11dfd6dbe9b5d89
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-25|00:25:30.961] New local node record                    seq=1,740,443,130,803 id=5c0f481e52bf6766 ip=10.1.0.178 udp=39529 tcp=38977
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:30.981Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/util.go:93	messenger started, public key: 0x040da8af244c48efedcce3e6d3c3ff3fa0059e8f99a00ee714878a2bd4596de3ce1a5d9869df28bd689db33753d4b39223a1a5d78d16b123c27c2c4d23a9e72f3e
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - 2025-02-25T00:25:31.972Z	INFO	0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135', '-kid', '0x5991c117b01d9f460b63cae2ec74abb9afd2b85661af889425db7acf2047c393']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - 2025-02-25T00:25:32.741Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - -name 0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - -key-uid 0x5991c117b01d9f460b63cae2ec74abb9afd2b85661af889425db7acf2047c393
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - 2025-02-25T00:25:32.741Z	INFO	0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-25|00:25:32.779] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - ERROR[02-25|00:25:32.785] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-25|00:25:32.786] New local node record                    seq=1,737,419,614,800 id=3120ffa1600d036e ip=127.0.0.1 udp=0 tcp=45055
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-25|00:25:32.786] Started P2P networking                   self="enode://1eee99e98b47a050566c4564713970fad84b9429798d00aea57ad2415987c1634a030183bee393717b626509c0bb811572a545295a0a6affb26c297bb7a1fbf9@127.0.0.1:45055?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-25|00:25:32.786] HTTP server started                      endpoint=127.0.0.1:6135 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-25|00:25:32.816] New local node record                    seq=1,740,443,132,792 id=f03c401e211344de ip=10.1.0.178 udp=59025 tcp=43743
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-25|00:25:32.818] New local node record                    seq=1,740,443,132,793 id=f03c401e211344de ip=10.1.0.178 udp=59025 tcp=43743
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - 2025-02-25T00:25:32.955Z	INFO	0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135	status-cli/util.go:62	existing account, key UID: 0x5991c117b01d9f460b63cae2ec74abb9afd2b85661af889425db7acf2047c393
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-25|00:25:32.956] New local node record                    seq=1,740,443,132,794 id=f03c401e211344de ip=10.1.0.178 udp=59025 tcp=43743
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - 2025-02-25T00:25:32.968Z	INFO	0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135	status-cli/util.go:93	messenger started, public key: 0x04890fe10a90ba0a415f5f05e7becd52b45f8f4c7c38236a549036dde17457d50811ab1bfb1b3144b4f859a3c323542ff810a716ef579e51035298bc110536cb3c
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-25|00:25:33.820] New local node record                    seq=1,740,443,128,802 id=750017867e0995b4 ip=172.171.254.176 udp=37743 tcp=31808
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - 2025-02-25T00:25:33.982Z	INFO	0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134', '-kid', '0xc74a6694789777559695ac9bf06c43cc94c5d4dd1cbd3c1d3eaabe55410855c4']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - 2025-02-25T00:25:34.749Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - -name 0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - -key-uid 0xc74a6694789777559695ac9bf06c43cc94c5d4dd1cbd3c1d3eaabe55410855c4
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - 2025-02-25T00:25:34.749Z	INFO	0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-25|00:25:34.788] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-25|00:25:34.793] New local node record                    seq=1,737,419,605,674 id=1e19dde297f8874d ip=127.0.0.1 udp=0 tcp=36447
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-25|00:25:34.793] Started P2P networking                   self="enode://99a2b3d7a793500a85f73e4baddc4643c65467df907e6eb7d569a6434b36f4a3255fc0d41038ce07a88508f57eb8e3613c00a996639de8c6b65ea83232116fd0@127.0.0.1:36447?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - ERROR[02-25|00:25:34.796] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-25|00:25:34.797] HTTP server started                      endpoint=127.0.0.1:6134 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-25|00:25:34.819] New local node record                    seq=1,740,443,134,800 id=706424cfff10250e ip=10.1.0.178 udp=44698 tcp=37775
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-25|00:25:34.823] New local node record                    seq=1,740,443,134,801 id=706424cfff10250e ip=10.1.0.178 udp=44698 tcp=37775
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - 2025-02-25T00:25:34.962Z	INFO	0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134	status-cli/util.go:62	existing account, key UID: 0xc74a6694789777559695ac9bf06c43cc94c5d4dd1cbd3c1d3eaabe55410855c4
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-25|00:25:34.963] New local node record                    seq=1,740,443,134,802 id=706424cfff10250e ip=10.1.0.178 udp=44698 tcp=37775
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - 2025-02-25T00:25:34.983Z	INFO	0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134	status-cli/util.go:93	messenger started, public key: 0x044da57de11adc9391ad5ca1677b0ff2da9e02f50f2184d27852966cb46e21a0759e1e0af6514a5d60dbec05dda5ab574f265a71a340caa3f98c92f9091f927a6d
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-25|00:25:35.813] New local node record                    seq=1,740,443,130,804 id=5c0f481e52bf6766 ip=172.171.254.176 udp=39529 tcp=31811
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:35.981Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139', '-kid', '0x98806521a617d0fb4d2140c292e22157c3d3197b212f81cd0a3bf164774580f0']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - 2025-02-25T00:25:36.747Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - -name 0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - -key-uid 0x98806521a617d0fb4d2140c292e22157c3d3197b212f81cd0a3bf164774580f0
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - 2025-02-25T00:25:36.747Z	INFO	0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-25|00:25:36.786] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - ERROR[02-25|00:25:36.792] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-25|00:25:36.792] New local node record                    seq=1,737,419,651,311 id=acb6477d150fec3c ip=127.0.0.1 udp=0 tcp=43815
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-25|00:25:36.792] Started P2P networking                   self="enode://437d152e7e8c57d86065f0879910c1f64ae6e14f007b9dc5b5c671ecfb463b66da4ffe7a8753f396f6232e58841a36808d6b65b1c05fb2a3aa9d8ed97ef447e0@127.0.0.1:43815?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-25|00:25:36.792] HTTP server started                      endpoint=127.0.0.1:6139 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-25|00:25:36.828] New local node record                    seq=1,740,443,136,804 id=59e50ca050a1f4e9 ip=10.1.0.178 udp=0 tcp=46007
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-25|00:25:36.832] New local node record                    seq=1,740,443,136,805 id=59e50ca050a1f4e9 ip=10.1.0.178 udp=44165 tcp=46007
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-25|00:25:36.833] New local node record                    seq=1,740,443,136,806 id=59e50ca050a1f4e9 ip=10.1.0.178 udp=44165 tcp=46007
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - 2025-02-25T00:25:36.975Z	INFO	0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139	status-cli/util.go:62	existing account, key UID: 0x98806521a617d0fb4d2140c292e22157c3d3197b212f81cd0a3bf164774580f0
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-25|00:25:36.975] New local node record                    seq=1,740,443,136,807 id=59e50ca050a1f4e9 ip=10.1.0.178 udp=44165 tcp=46007
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - 2025-02-25T00:25:36.998Z	INFO	0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139	status-cli/util.go:93	messenger started, public key: 0x040d9cf92cec5a7f9e887525cd79bfc4a927af9e1ce7c4d89246d974613d6ea084c360b2bd11007d57e296de08a6fb0dd0d813d28e57cf5f797745241ddabdd973
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-25|00:25:37.810] New local node record                    seq=1,740,443,132,795 id=f03c401e211344de ip=172.171.254.176 udp=59025 tcp=31812
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - 2025-02-25T00:25:37.968Z	INFO	0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132', '-kid', '0x317d1096e362af7561ea2f494e3cf3f715137ba7da24d801eea4ac7e10895318']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - 2025-02-25T00:25:38.749Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - -name 0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - -key-uid 0x317d1096e362af7561ea2f494e3cf3f715137ba7da24d801eea4ac7e10895318
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - 2025-02-25T00:25:38.749Z	INFO	0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-25|00:25:38.789] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-25|00:25:38.796] New local node record                    seq=1,737,419,587,417 id=a3e6d7f2cfadb3c4 ip=127.0.0.1 udp=0 tcp=36935
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-25|00:25:38.796] Started P2P networking                   self="enode://aad0dc447a381566b4aa3dc121ad9ba5674e454f820fcda769e16c748042a1e278d4c98d22a4e518f933d8135d897befd8ecfe65c0f92930c9606e794e24e381@127.0.0.1:36935?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - ERROR[02-25|00:25:38.796] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-25|00:25:38.797] HTTP server started                      endpoint=127.0.0.1:6132 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-25|00:25:38.821] New local node record                    seq=1,740,443,128,803 id=750017867e0995b4 ip=172.171.254.176 udp=37743 tcp=31809
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-25|00:25:38.834] New local node record                    seq=1,740,443,138,809 id=f19a089af31dd6ac ip=10.1.0.178 udp=34781 tcp=41671
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-25|00:25:38.839] New local node record                    seq=1,740,443,138,810 id=f19a089af31dd6ac ip=10.1.0.178 udp=34781 tcp=41671
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - WARN [02-25|00:25:38.975] Served personal_ecRecover                reqid=1 duration="51.144µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - WARN [02-25|00:25:38.977] Served personal_ecRecover                reqid=2 duration="22.843µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - 2025-02-25T00:25:38.981Z	INFO	0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132	status-cli/util.go:62	existing account, key UID: 0x317d1096e362af7561ea2f494e3cf3f715137ba7da24d801eea4ac7e10895318
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-25|00:25:38.981] New local node record                    seq=1,740,443,138,811 id=f19a089af31dd6ac ip=10.1.0.178 udp=34781 tcp=41671
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - 2025-02-25T00:25:39.005Z	INFO	0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132	status-cli/util.go:93	messenger started, public key: 0x041389374bcaf9a1c36230942c58f4e3968c0a2472bfa9041916414f11d228a3b0605c6ede03ff65463fe8a22126d1f2c750991f03bdf352eeb6442611e2b87aa1
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - WARN [02-25|00:25:39.975] Served personal_ecRecover                reqid=3 duration="34.314µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - 2025-02-25T00:25:39.984Z	INFO	0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138', '-kid', '0x1226dcff3cb70a51c0f10a4a47de02a888068b914e654e5626095edc6edf1707']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - 2025-02-25T00:25:40.750Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - -name 0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - -key-uid 0x1226dcff3cb70a51c0f10a4a47de02a888068b914e654e5626095edc6edf1707
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - 2025-02-25T00:25:40.750Z	INFO	0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-25|00:25:40.790] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-25|00:25:40.796] New local node record                    seq=1,737,419,642,180 id=e3228c0ebd0ad048 ip=127.0.0.1 udp=0 tcp=36053
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-25|00:25:40.796] Started P2P networking                   self="enode://e1e8dc76505064a0cd4888bcc75868421543cdfbbcdbe271fd2f512ec4e0013e1e8829edf9dfcf895198aad4056c5e9c238c1c63eab627cfd4fa85b5908c1664@127.0.0.1:36053?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - ERROR[02-25|00:25:40.797] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-25|00:25:40.798] HTTP server started                      endpoint=127.0.0.1:6138 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-25|00:25:40.813] New local node record                    seq=1,740,443,130,805 id=5c0f481e52bf6766 ip=172.171.254.176 udp=39529 tcp=31810
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-25|00:25:40.833] New local node record                    seq=1,740,443,140,812 id=67129609f0f1b964 ip=10.1.0.178 udp=0 tcp=46705
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-25|00:25:40.834] New local node record                    seq=1,740,443,140,813 id=67129609f0f1b964 ip=10.1.0.178 udp=52829 tcp=46705
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-25|00:25:40.836] New local node record                    seq=1,740,443,140,814 id=67129609f0f1b964 ip=10.1.0.178 udp=52829 tcp=46705
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - 2025-02-25T00:25:40.980Z	INFO	0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138	status-cli/util.go:62	existing account, key UID: 0x1226dcff3cb70a51c0f10a4a47de02a888068b914e654e5626095edc6edf1707
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - INFO [02-25|00:25:40.980] New local node record                    seq=1,740,443,140,815 id=67129609f0f1b964 ip=10.1.0.178 udp=52829 tcp=46705
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - 2025-02-25T00:25:41.003Z	INFO	0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138	status-cli/util.go:93	messenger started, public key: 0x04c05ffc7fbe973116e8a7017bbd49632c7aaa31715924eb62497ce229bd52a6ffe6904b99dd534f66d4b1364770dce92fcc97a901a1adb98dbebddaf6bd876eb6
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:25:41.812] New local node record                    seq=1,740,443,126,800 id=a6f2f22fe1d4bca5 ip=172.171.254.176 udp=54602 tcp=31808
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-25|00:25:41.826] New local node record                    seq=1,740,443,136,808 id=59e50ca050a1f4e9 ip=172.171.254.176 udp=44165 tcp=31812
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - 2025-02-25T00:25:42.000Z	INFO	0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136', '-kid', '0x81bb8c3cf7199471bd4b3dda97fca44a8d2a482f9cda5e413efb11fa68db3f52']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - 2025-02-25T00:25:42.751Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - -name 0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - -key-uid 0x81bb8c3cf7199471bd4b3dda97fca44a8d2a482f9cda5e413efb11fa68db3f52
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - 2025-02-25T00:25:42.751Z	INFO	0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-25|00:25:42.803] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - ERROR[02-25|00:25:42.809] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-25|00:25:42.809] New local node record                    seq=1,737,419,623,946 id=877b25ac259fbb61 ip=127.0.0.1 udp=0 tcp=35463
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-25|00:25:42.809] Started P2P networking                   self="enode://3c43eceef4f4dee8fd369a60b6400d45eefc85a84579959675a0d624fbbe868f4dbc522c79a78bf55c7844278981e3bd2bbe4ed39727170d2035215ea236eb0c@127.0.0.1:35463?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-25|00:25:42.810] HTTP server started                      endpoint=127.0.0.1:6136 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-25|00:25:42.852] New local node record                    seq=1,740,443,142,827 id=629626e88094058a ip=10.1.0.178 udp=39792 tcp=43305
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-25|00:25:42.853] New local node record                    seq=1,740,443,142,828 id=629626e88094058a ip=10.1.0.178 udp=39792 tcp=43305
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - 2025-02-25T00:25:43.003Z	INFO	0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136	status-cli/util.go:62	existing account, key UID: 0x81bb8c3cf7199471bd4b3dda97fca44a8d2a482f9cda5e413efb11fa68db3f52
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - INFO [02-25|00:25:43.004] New local node record                    seq=1,740,443,142,829 id=629626e88094058a ip=10.1.0.178 udp=39792 tcp=43305
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - 2025-02-25T00:25:43.026Z	INFO	0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136	status-cli/util.go:93	messenger started, public key: 0x042898fe963497a6d5cdf4bae695090aae18993c81f12e39be2b51b3980b806fa22fd3688e25d44663ce4c02010d7d8856c7bff8c610d1c0d3be4f2d1187426bfe
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X035CD90EF0ACDD7CA1173984B56C36A1D9A0119D00F787C3B99BC9705B6D394094_6137 - INFO [02-25|00:25:43.821] New local node record                    seq=1,740,443,128,804 id=750017867e0995b4 ip=172.171.254.176 udp=37743 tcp=31808
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - 2025-02-25T00:25:44.005Z	INFO	0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132	status-cli/serve.go:107	retrieve messages...
[32mINFO    [0m src.node.status_node:status_node.py:52 Serving node with command: ['./status-cli', 'serve-account', '-n', '0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131', '-kid', '0xe171a00735b54dcf9ecb7a27904fdd4b143362df5354c1e469753d5f08023459']
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 2 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - 2025-02-25T00:25:44.750Z	INFO	status-cli/serve.go:34	Running serve-account command, with:
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - -name 0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - -key-uid 0xe171a00735b54dcf9ecb7a27904fdd4b143362df5354c1e469753d5f08023459
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - -interactive false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - -add
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - -debug false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - -help false
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - 
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - 2025-02-25T00:25:44.751Z	INFO	0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131	status-cli/util.go:55	starting messenger
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-25|00:25:44.793] Starting peer-to-peer node               instance=StatusIM/linux-amd64/go1.22.3
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-25|00:25:44.798] New local node record                    seq=1,737,419,578,301 id=91b1bc7236045679 ip=127.0.0.1 udp=0 tcp=44809
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-25|00:25:44.798] Started P2P networking                   self="enode://91d8c4c0322908732547f0a21fe21a53e4a636758e1615a385e922f834ad499c271c69dcee9a8fefe3c7479c1584e71ead9adf547235b9547d0349db0b06bee9@127.0.0.1:44809?discport=0"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - ERROR[02-25|00:25:44.800] Unavailable modules in HTTP API list     unavailable=[wakuv2] available="[admin debug web3 eth rpcstats appmetrics appgeneral peer personal status pending ens communitytokens stickers updates settings accounts multiaccounts browsers permissions mailservers gif chat wallet waku wakuext localnotifications]"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-25|00:25:44.801] HTTP server started                      endpoint=127.0.0.1:6131 auth=false prefix= cors= vhosts=
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-25|00:25:44.817] New local node record                    seq=1,740,443,134,803 id=706424cfff10250e ip=172.171.254.176 udp=44698 tcp=31813
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-25|00:25:44.835] New local node record                    seq=1,740,443,144,811 id=3079b466131bee6b ip=10.1.0.178 udp=58863 tcp=35607
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-25|00:25:44.838] New local node record                    seq=1,740,443,144,812 id=3079b466131bee6b ip=10.1.0.178 udp=58863 tcp=35607
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - 2025-02-25T00:25:44.999Z	INFO	0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131	status-cli/util.go:62	existing account, key UID: 0xe171a00735b54dcf9ecb7a27904fdd4b143362df5354c1e469753d5f08023459
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - INFO [02-25|00:25:45.000] New local node record                    seq=1,740,443,144,813 id=3079b466131bee6b ip=10.1.0.178 udp=58863 tcp=35607
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - 2025-02-25T00:25:45.023Z	INFO	0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131	status-cli/util.go:93	messenger started, public key: 0x047171e21124210e88fa7e404355815ceac9793d65706770d642813fdfaacc4e0def04dd42a5be6f28ef68d9e76a297cb7f6fe0326d50547faffaffbaf0fd1a662
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - INFO [02-25|00:25:45.814] New local node record                    seq=1,740,443,130,806 id=5c0f481e52bf6766 ip=172.171.254.176 udp=39529 tcp=31811
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X021B95D970E2773934F546C0B698AAAF3E7754AB90CEC96EB720B66FAE1120421E_6138 - 2025-02-25T00:25:46.003Z	INFO	0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.libs.common:common.py:15 Sleeping for 4 seconds
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X039E80917B3BFD4803FE7803AE300A62C373A7E0BE767C16178980884A56208467_6136 - 2025-02-25T00:25:48.026Z	INFO	0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - WARN [02-25|00:25:48.199] Served personal_ecRecover                reqid=1 duration="51.246µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - WARN [02-25|00:25:48.200] Served personal_ecRecover                reqid=2 duration="17.843µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - WARN [02-25|00:25:48.201] Served personal_ecRecover                reqid=3 duration="17.412µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - WARN [02-25|00:25:48.203] Served personal_ecRecover                reqid=4 duration="20.588µs" err="signature must be 65 bytes long"
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 30e7b46a-1634-4882-867c-1c5fbbf1d601_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_4 (ID=0xd010664918963e61817beede53677fc03831911158076570924090f1da25c0df)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: a203d1f6-c143-46e7-8a21-310d7255e629_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_2 (ID=0xf3658d26160a6b4dde83f733792ebc2ccaae1b444d8a5ee76df77b8c6ef13aef)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 1374e452-3afd-4372-bab2-797127013dda_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_16 (ID=0x216600891d4c133cdb05f3ba13c34e7652d7ace734485eb1798b45c88567b9fe)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 4a71aa7b-3d8f-4c76-832c-f90872d9ff8f_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_14 (ID=0xda2e4033152227fdc72d160e8a93d5dda7293c3d101ea30811aef79e08d2538b)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 84a1c37c-6311-463d-95fa-33c46a068092_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_12 (ID=0x27c6bc79728b26688cd29d519faca962c467df84e0ad46bd190226460d68f9f1)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 40ee2073-0b31-4b01-98fa-b75c04218b89_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_10 (ID=0xbd7c89aedbad700461dfecf2d89dcf0527650510864ec238cdd14d0476954788)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 875980f3-489a-4024-a319-301e42591f58_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_8 (ID=0xa95ef99dcf1e54f00e4039ac50c25a0f9fed196fe7da5e45196275ef359692cd)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: 9131e5be-39e7-4523-9e6f-ee1dbbe22b22_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_24 (ID=0xf3e2e7b519c4269d77ba49db500b47ff1c86bc886ade43eca8e94cb8adf29ac2)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: ac8f08ce-e3ad-4b03-95ea-e22c52a2c924_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_20 (ID=0x3b0901359071b49a23406434c1825b60f35e68e365cf9ff5893472aa0e5fae3a)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: ea0bdffa-97c1-4b6d-a004-cb82bae59152_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_18 (ID=0xeb67118e087d27e6585aae589b47b84036de0c929e8e035d8c8a060263b09b34)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: e2d462b8-43e8-4d9c-a251-26c1c1cf3dcb_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_6 (ID=0xab96fb408bd1821d0229d55e181a1803367a3817b882a9a511c60a310b88735a)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: c809dbaf-2af4-4554-b083-d1787cd3564e (ID=0x59479af78f5dccfe1be0a507ae215fb38e26a418072439d2f945f29d67bc531c)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: bc140d4b-33bf-4fb6-b1db-8a859796aa3c_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_22 (ID=0xf94004c2310b12bb8db1de41c7d0674814c998e05d0f1970e7de073f10d3bd36)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X03084787B0EB1391FD622FDA8FEE5F9FDA3571467EE2C4DBCCE9D10E4CB50442AD_6133 - 2025-02-25T00:25:48.219Z	INFO	0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133	status-cli/serve.go:70	message received: f66e0ac0-b15a-40a2-a65d-f6f5e67534f3_message_from_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133_0 (ID=0xbf1fdbd903aebb62ad5feb4bdcc2bc7700d53cfd920426ab2ba47330d7972be9)
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-25|00:25:48.728] New local node record                    seq=1,740,443,132,796 id=f03c401e211344de ip=10.1.0.178 udp=59025 tcp=43743
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-25|00:25:48.829] New local node record                    seq=1,740,443,138,812 id=f19a089af31dd6ac ip=172.171.254.176 udp=34781 tcp=31815
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-25|00:25:48.961] New local node record                    seq=1,740,443,132,797 id=f03c401e211344de ip=10.1.0.178 udp=59025 tcp=43743
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-25|00:25:48.963] New local node record                    seq=1,740,443,132,798 id=f03c401e211344de ip=10.1.0.178 udp=59025 tcp=43743
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X036DDC1C91806EEFCFBD8EB1ACCADC7EFF82D79F1FED6E8FCCD08F9CF737D02C1F_6131 - 2025-02-25T00:25:50.024Z	INFO	0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131	status-cli/serve.go:107	retrieve messages...
[35mDEBUG   [0m src.steps.common:common.py:39 Entering context manager: add_latency
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:39434 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:25:51.813] New local node record                    seq=1,740,443,126,801 id=a6f2f22fe1d4bca5 ip=172.171.254.176 udp=54602 tcp=31809
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:25:56.812] New local node record                    seq=1,740,443,126,802 id=a6f2f22fe1d4bca5 ip=172.171.254.176 udp=54602 tcp=31808
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02D815E00131E404D7DE053208ED6DB77A996403243C52BF8B20922A5E947BBD43_6135 - INFO [02-25|00:25:57.811] New local node record                    seq=1,740,443,132,799 id=f03c401e211344de ip=172.171.254.176 udp=59025 tcp=31812
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-25|00:25:58.828] New local node record                    seq=1,740,443,138,813 id=f19a089af31dd6ac ip=10.1.0.178 udp=34781 tcp=41671
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-25|00:25:59.816] New local node record                    seq=1,740,443,134,804 id=706424cfff10250e ip=10.1.0.178 udp=44698 tcp=37775
[31m[1mERROR   [0m src.node.rpc_client:rpc_client.py:32 Request to http://127.0.0.1:39434 usig method wakuext_fetchCommunity timed out after 10 seconds. Node: first_node
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:39434 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02E2DAF7144BE6E5FEF22E6EA21F04302A51CBA97EBE58C010478D8EAC1AEE62C6_6134 - INFO [02-25|00:26:04.816] New local node record                    seq=1,740,443,134,805 id=706424cfff10250e ip=172.171.254.176 udp=44698 tcp=31813
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02195F3912BCA1C5B6EA510D10A67F6C6AEBA77702A48591A9C98240931603971C_6132 - INFO [02-25|00:26:08.829] New local node record                    seq=1,740,443,138,814 id=f19a089af31dd6ac ip=172.171.254.176 udp=34781 tcp=31812
[31m[1mERROR   [0m src.node.rpc_client:rpc_client.py:32 Request to http://127.0.0.1:39434 usig method wakuext_fetchCommunity timed out after 10 seconds. Node: first_node
[35mDEBUG   [0m src.node.rpc_client:rpc_client.py:21 Node: first_node sends request at address: http://127.0.0.1:39434 with payload: {"jsonrpc": "2.0", "method": "wakuext_fetchCommunity", "params": [{"communityKey": "0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2", "waitForResponse": true, "tryDatabase": true}], "id": 1}
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:26:11.813] New local node record                    seq=1,740,443,126,803 id=a6f2f22fe1d4bca5 ip=172.171.254.176 udp=54602 tcp=31808
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02C805D0BC6C60EAE2DF62A9780B0B46B75301283075057ECDC677A2DC7A3CBFE1_6139 - INFO [02-25|00:26:11.826] New local node record                    seq=1,740,443,136,809 id=59e50ca050a1f4e9 ip=172.171.254.176 udp=44165 tcp=31812
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-25|00:26:13.511] New local node record                    seq=1,740,443,120,944 id=c96ac776770817a1 ip=10.1.0.178 udp=32967 tcp=34983
[35mDEBUG   [0m src.node.status_node:status_node.py:65 0X02CBC80D3F2C70DE89BC14DF08CD90AF4D038A1AC46CA994A1C1DBAFEFF4BD46B2_6130 - INFO [02-25|00:26:16.812] New local node record                    seq=1,740,443,126,804 id=a6f2f22fe1d4bca5 ip=172.171.254.176 udp=54602 tcp=31808
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-25|00:26:17.039] New local node record                    seq=1,740,443,120,945 id=c96ac776770817a1 ip=10.1.0.178 udp=32967 tcp=34983
[35mDEBUG   [0m src.node.status_node:status_node.py:65 FIRST_NODE - INFO [02-25|00:26:17.041] New local node record                    seq=1,740,443,120,946 id=c96ac776770817a1 ip=10.1.0.178 udp=32967 tcp=34983
[31m[1mERROR   [0m src.node.rpc_client:rpc_client.py:32 Request to http://127.0.0.1:39434 usig method wakuext_fetchCommunity timed out after 10 seconds. Node: first_node
[35mDEBUG   [0m src.steps.common:common.py:44 Exiting context manager: add_latency
[35mDEBUG   [0m tests.conftest:conftest.py:42 Running fixture teardown: save_node_logs_on_fail
[35mDEBUG   [0m tests.conftest:conftest.py:45 Test failed, attempting to rename and move logs to the test_results folder
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file first_node.log to test_results/testfetchcommunity_test_fetch_community_with_latency_first_node.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139.log
[35mDEBUG   [0m tests.conftest:conftest.py:55 Moved log file 0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134.log to test_results/testfetchcommunity_test_fetch_community_with_latency_0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134.log
[35mDEBUG   [0m tests.conftest:conftest.py:62 Running fixture teardown: clear_open_nodes
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: first_node
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02cbc80d3f2c70de89bc14df08cd90af4d038a1ac46ca994a1c1dbafeff4bd46b2_6130
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x035cd90ef0acdd7ca1173984b56c36a1d9a0119d00f787c3b99bc9705b6d394094_6137
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x03084787b0eb1391fd622fda8fee5f9fda3571467ee2c4dbcce9d10e4cb50442ad_6133
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02d815e00131e404d7de053208ed6db77a996403243c52bf8b20922a5e947bbd43_6135
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02e2daf7144be6e5fef22e6ea21f04302a51cba97ebe58c010478d8eac1aee62c6_6134
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02c805d0bc6c60eae2df62a9780b0b46b75301283075057ecdc677a2dc7a3cbfe1_6139
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x02195f3912bca1c5b6ea510d10a67f6c6aeba77702a48591a9c98240931603971c_6132
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x021b95d970e2773934f546c0b698aaaf3e7754ab90cec96eb720b66fae1120421e_6138
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x039e80917b3bfd4803fe7803ae300a62c373a7e0be767c16178980884a56208467_6136
[32mINFO    [0m src.node.status_node:status_node.py:73 Stopping node with name: 0x036ddc1c91806eefcfbd8eb1accadc7eff82d79f1fed6e8fccd08f9cf737d02c1f_6131